src/components/utils/onfidoApi.js

```diff
@@ -46,9 +46,12 @@ export const requestChallenges = (token, onSuccess, onError) => {
   performHttpReq(options, onSuccess, onError)
 }
 
-const objectToFormData = (object) => {
+const objectToFormData = ({filename, blob, ...other}) => {
```

Comment:
hmm, there is a bit of a risk here is someone wants to send an object with both a key file and a key filename. I think we could avoid that if we wrap in one object which of the type {blob:Blob,filename:string}. The `if` could figure out if that object is of that type and then do the logic `formData.append('file', blob, filename)`

Reply:
Good point!